rpcs3/Emu/RSX/D3D12/D3D12Formats.cpp

```diff
@@ -110,7 +110,7 @@ D3D12_STENCIL_OP get_stencil_op(u32 op) noexcept
 	case CELL_GCM_INCR: return D3D12_STENCIL_OP_INCR;
 	case CELL_GCM_DECR: return D3D12_STENCIL_OP_DECR;
 	case CELL_GCM_INVERT: return D3D12_STENCIL_OP_INVERT;
-	case CELL_GCM_INCR_WRAP:
+	case CELL_GCM_INCR_WRAP: return D3D12_STENCIL_OP_INCR;
```

Comment:
@Zangetsu38 "CELL_GCM_INCR_WRAP" and "CELL_GCM_INCR" clearly do different things and if you think "using a fallback is better than nothing" then you should at least note somewhere that it is a hacky fallback and not the correct behavior.

Also "Euu" is not an English word or onomatopoeia.

Anyway just skimming the docs it seems to me that
CELL_GCM_INCR = D3D12_STENCIL_OP_INCR_SAT
CELL_GCM_DECR = D3D12_STENCIL_OP_DECR_SAT
CELL_GCM_INCR_WRAP = D3D12_STENCIL_OP_INCR
CELL_GCM_DECR_WRAP = D3D12_STENCIL_OP_DECR

but @vlj should probably check that